Scripts/script-ParseCSV.yml

```diff
@@ -271,6 +271,10 @@ args:
   - "no"
   description: parses and converts all the rows in csv into json and puts into context.
   defaultValue: "yes"
+- name: codec
+  description: The codec type used to parse the file (some character sets are not
```

Comment:
please let @kirbles19 review the description

Reply:
Contacted him now.